archaius2-test/src/test/java/com/netflix/archaius/test/Archaius2RuleTest.java

```diff
@@ -0,0 +1,54 @@
+package com.netflix.archaius.test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+@TestPropertyOverride({"classLevelProperty=present", "cascading=type"})
+public class Archaius2RuleTest extends Archaius2RuleTestParentTest {
+    
+    @Rule
+    public Archaius2TestConfig config = new Archaius2TestConfig();
+    
+    @Test 
+    @TestPropertyOverride({"testLevelProperty=present"})
+    public void testBasicPropertyResolution() {
+        assertNotNull(config);
+        assertEquals("present", config.getString("parentClassLevelProperty"));
+        assertEquals("present", config.getString("classLevelProperty"));
+        assertEquals("present", config.getString("testLevelProperty"));
```

Comment:
Consider adding an assert to verify that the value of cascading is "type".  You never actually verify that multiple property values are parsed correctly. :-/